v3/src/components/graph/adornments/univariate-measures/univariate-measure-adornment-model.ts

```diff
@@ -71,13 +78,18 @@ export const UnivariateMeasureAdornmentModel = AdornmentModel
       const columnCount = topCatCount * xCatCount
       const rowCount = rightCatCount * yCatCount
       const totalCount = rowCount * columnCount
-      const attrId = xAttrId ? xAttrId : yAttrId
+      const attrId = xAttrId && dataConfig.attributeType("x") ? xAttrId : yAttrId
       for (let i = 0; i < totalCount; ++i) {
         const cellKey = self.setCellKey(options, i)
-        const instanceKey = self.instanceKey(cellKey)
+        // If there are no cases in the cell, do not add an adornment
+        if (dataConfig.subPlotCases(cellKey).length === 0) continue
```

Comment:
As mentioned previously, it may be better to add the adornment in this case and just not render it if there are no cases, but I'm also fine with leaving this as is and crossing that bridge if/when we come to it.

Reply:
Sorry -- I removed the `if (!Number.isFinite(value)) continue` line previously, but for some reason didn't think to remove this one, too. I've removed it in the latest commit.